addons/l10n_es_edi_tbai/models/account_edi_format.py

```diff
@@ -632,7 +632,7 @@ def _l10n_es_tbai_get_in_invoice_values_batuz(self, invoice):
         values['is_refund'] = invoice.move_type == 'in_refund'
         if values['is_refund']:
             values['credit_note_code'] = invoice.l10n_es_tbai_refund_reason
-            values['credit_note_invoice'] = invoice.reversed_entry_id
+            values['credit_note_invoices'] = invoice.reversed_entry_id
```

Comment:
```suggestion
            values['credit_note_invoice'] = values['credit_note_invoices'] = invoice.reversed_entry_id
```
Have you tested in stable without upgrading the module ?

I'm afraid that people who won't upgrade their module won't have the new template so the `t-if="credit_note_invoices"` will fail because it doesn't exist anymore

Reply:
I solved it in the last update. Significant changes are done completely in the new module.